Userland/Libraries/LibWeb/HTML/WorkerGlobalScope.h

```diff
@@ -28,12 +29,18 @@ namespace Web::HTML {
 // https://html.spec.whatwg.org/multipage/workers.html#the-workerglobalscope-common-interface
 // WorkerGlobalScope is the base class of each real WorkerGlobalScope that will be created when the
 // user agent runs the run a worker algorithm.
-class WorkerGlobalScope : public DOM::EventTarget {
+class WorkerGlobalScope
+    : public DOM::EventTarget
+    , public WindowOrWorkerGlobalScopeMixin {
     WEB_PLATFORM_OBJECT(WorkerGlobalScope, DOM::EventTarget);
 
 public:
     virtual ~WorkerGlobalScope() override;
 
+    // ^WindowOrWorkerGlobalScopeMixin
+    virtual Bindings::PlatformObject& this_impl() override { return *this; }
+    virtual Bindings::PlatformObject const& this_impl() const override { return *this; }
```

Comment:
Is there a reason we don't want to use CRTP here to get access to the "real type" ? Otherwise these changes lgtm.

Reply:
Only me not being familiar with C++ shenanigans like this & copying what I know once worked already 😛
Will update, thanks!